python/cudf/cudf/_fuzz_testing/tests/fuzz_test_orc.py

```diff
@@ -0,0 +1,70 @@
+# Copyright (c) 2020, NVIDIA CORPORATION.
+
+import io
+import sys
+
+import cudf
+from cudf._fuzz_testing.main import pythonfuzz
+from cudf._fuzz_testing.orc import OrcReader, OrcWriter
+from cudf._fuzz_testing.utils import (
+    ALL_POSSIBLE_VALUES,
+    compare_dataframe,
+    run_test,
+)
+
+
+@pythonfuzz(
+    data_handle=OrcReader,
+    params={
+        "columns": ALL_POSSIBLE_VALUES,
+        "skiprows": ALL_POSSIBLE_VALUES,
+        "num_rows": ALL_POSSIBLE_VALUES,
+    },
+)
+def orc_reader_test(input_tuple, skiprows, columns, num_rows):
+    # TODO: Remove skiprows=0 after
+    # following issue is fixed:
+    # https://github.com/rapidsai/cudf/issues/6563
+    skiprows = 0
+
+    pdf, parquet_buffer = input_tuple
+    expected_pdf = pdf[skiprows:]
+    if num_rows is not None:
+        expected_pdf = expected_pdf.head(num_rows)
+    if skiprows is not None or num_rows is not None:
+        expected_pdf = expected_pdf.reset_index(drop=True)
+    if columns is not None:
+        expected_pdf = expected_pdf[columns]
+
+    gdf = cudf.read_orc(
+        parquet_buffer, columns=columns, skiprows=skiprows, num_rows=num_rows
+    )
+
+    compare_dataframe(expected_pdf, gdf)
+
+
+@pythonfuzz(
+    data_handle=OrcWriter,
+    params={
+        "compression": [None, "snappy"],
```

Comment:
Is zlib compression not supported ? or None -> zlib ?

Reply:
Picked those parameters based on cython/python code 

https://github.com/rapidsai/cudf/blob/2df2f3a5658c520e11e39607f7fa24995c79dba7/python/cudf/cudf/_lib/orc.pyx#L93-L100

It appears we error for zlib currently